WMIAdventure/backend/WMIAdventure_backend/cards/migrations/0005_cardeffect_has_modifier.py

```diff
@@ -1,35 +1,39 @@
 # Generated by Django 3.2 on 2021-05-24 17:47
 
 from django.db import migrations, models
-
-from cards.models import CardEffect, CardLevel
+from cards.models import CardLevel as ModelCardLevel
+from cards.models import CardEffect as ModelCardEffect
 
 
 def insert_card_effects_values_to_db(apps, schema_editor):
+    CardEffect = apps.get_model('cards', 'CardEffect')
     CardEffect.objects.all().delete()
-    for value in CardEffect.EffectId.values:
-        item = CardEffect.objects.create(id=value)
+    for e in ModelCardEffect.EffectId:
+        item = CardEffect.objects.create(id=e.value,
+                                         name=e.label)
         item.save()
 
     # Modifying has_modifier values for some effects
     # One may also use views to modify them, we just set them here initially.
-    def change_modifier_true(effect: CardEffect.EffectId):
+    def change_modifier_true(effect: ModelCardEffect.EffectId):
         effect = CardEffect.objects.get(pk=effect.value)
         effect.has_modifier = True
         effect.save()
 
-    change_modifier_true(CardEffect.EffectId.DMG)
-    change_modifier_true(CardEffect.EffectId.SHIELD)
-    change_modifier_true(CardEffect.EffectId.HEAL)
+    change_modifier_true(ModelCardEffect.EffectId.DMG)
+    change_modifier_true(ModelCardEffect.EffectId.SHIELD)
+    change_modifier_true(ModelCardEffect.EffectId.HEAL)
 
 
 def insert_card_levels_values_to_db(apps, schema_editor):
     """
     Inserts all possible CardLevel records to database.
     """
+    CardLevel = apps.get_model('cards', 'CardLevel')
+    Level = ModelCardLevel.Level
```

Comment:
Tak samo jak poprzednie pytanie, po co podwójny import?

Reply:
Bo musimy zaimportować Level i EffectId. To są klasy Pythowe a nie da się ich wyciągnąć z tego z app